### PR TITLE
Fix keploy gives error for gin mongo when recorded with docker compose

### PR DESCRIPTION
**Pull request by sagnik3788**
Closes: #2038 

#### Describe the changes you've made
Remove the indocker flag 

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

#### Describe if there is any unusual behaviour of your code
NA

### CODE REVIEW

**Comment by sagnik3788** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by sagnik3788** [comment]
Is it going to work fine for the Docker Compose  error in the samples-go/gin-mongo ?
@shivamsouravjha

**Review by shivamsouravjha — APPROVED** [review]
LGTM!

**Comment by shivamsouravjha** [comment]
Hey @sagnik3788 please sign the CLA.

**Comment by sagnik3788** [comment]
I have read the CLA Document and I hereby sign the CLA

File: config/default.go [hunk 1]
```diff
@@ -61,7 +60,6 @@ const InternalConfig = `
 enableTesting: false
 keployContainer: "keploy-v2"
 keployNetwork: "keploy-network"
-inDocker: false
 cmdType: "native"
```
Comment (shivamsouravjha) [inline config/default.go:63]: you shouldn't remove it from the internal config.

**Review by shivamsouravjha — APPROVED** [review]
LGTM!

**Pull request merged**